tasks/src/sentence-similarity/data.ts

```diff
@@ -20,23 +20,23 @@ const taskData: TaskData = {
 
 			
 			{
-				label:   "Source Sentence",
+				label:   "Main sentence",
 				content: "Machine learning is so easy.",
 				type:    "text",
 			},
 			{
 				label:   "Sentences to compare to",
-				content: "Deep learning is so effortless.",
+				content: "Deep learning is so straightforward.",
```

Comment:
You can update this example, just make sure to be consistent.

Reply:
Sure. I tried to keep the message.